ci/do_ci.sh

```diff
@@ -463,11 +463,12 @@ case $CI_TARGET in
             --define log_debug_assert_in_release=disabled
         echo "Building binary with wasm=wasmtime... and logging disabled"
         bazel build "${BAZEL_BUILD_OPTIONS[@]}" \
+            --action_env=TEST=TEST \
             --config=compile-time-options \
             --define wasm=wasmtime \
             --define enable_logging=disabled \
             -c fastbuild \
-            @envoy//source/exe:envoy-static \
+            @envoy//contrib/exe:envoy-static \
```

Comment:
we kinda want the test with the 2 following conditions to compare:

- existing - ie `//source`
- proposed change - ie _both_

Reply:
ok, let me try to build those two in c-t-o

Reply:
cool - was wondering whether we would need to separate as you have done

the benefit of this is that we only need to run the ci once to get the timings, and can see exactly how long the contrib build takes

Reply:
I run two bazel cmds, suppose we can get the timings separately. Let me trigger the CI it seems the CI broken friday.